src/pages/Cart/Cart.js

```diff
@@ -1,8 +1,154 @@
-import React from 'react'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import Count from './Count/Count'
+import './Cart.scss'
 
 const Cart = () => {
-  return <h1>hello</h1>
+  const navigate = useNavigate()
+
+  const [checkItems, setCheckItems] = useState([])
+  const [quantity, setQuantity] = useState({})
+
+  let totalPrice = 10000
+  let deliveryPrice = 3000
+
+  const handleCheck = (checked, id) => {
+    if (checked) {
+      setCheckItems(prev => [...prev, id])
+    } else {
+      setCheckItems(checkItems.filter(item => item !== id))
+    }
+  }
+
+  const handleAllCheck = checked => {
+    if (checked) {
+      setCheckItems(CART_ITEM_LIST.map(item => item.id))
+    } else {
+      setCheckItems([])
+    }
+  }
+
+  const handleQuantityChange = (id, value) => {
+    setQuantity(prev => ({ ...prev, [id]: value }))
+  }
+
+  return (
+    <div className="cart">
+      <div className="cartHeader">
+        <h1>장바구니</h1>
+      </div>
+      <main className="cartContainer">
+        <section className="cartProduct">
+          <div className="cartProductOption">
+            <div>
+              <input
+                type="checkbox"
+                onChange={e => handleAllCheck(e.target.checked)}
+                checked={checkItems.length === CART_ITEM_LIST.length}
+              />
+              <span>전체선택</span>
+              <span className="checkedItems">
+                ({checkItems.length}/{CART_ITEM_LIST.length})
+              </span>
+            </div>
+            <button
+              className="cartProductDeleteBtn"
+              onClick={() => {
+                alert('선택상품을 지우시겠습니까?')
+              }}
+            >
+              선택삭제
+            </button>
+          </div>
+          <ul className="cartProductList">
+            {CART_ITEM_LIST.map((item, index) => {
+              return (
+                <li className="cartProductItem" key={index}>
+                  <input
+                    type="checkbox"
+                    onChange={e => handleCheck(e.target.checked, item.id)}
+                    checked={checkItems.includes(item.id) ? true : false}
```

Comment:
```suggestion
                    checked={checkItems.includes(item.id)}
```
조건 자체가 boolean 값이기 때문에 삼항연산자를 사용하지 않으셔도 됩니다!